napari/layers/points/_points_key_bindings.py

```diff
@@ -12,6 +15,10 @@ def register_points_action(description):
     return register_layer_action(Points, description)
 
 
+def register_label_mode_action(description):
+    return register_label_attr_action(Points, description, 'mode')
```

Comment:
Uhm, there seems to be a copy-pase issue here, it should probably be `points`, noy `label`. (And same for all the following)

Reply:
it should be `layer`, not `label` in every place